x-pack/plugins/canvas/canvas_plugin_src/functions/common/sort.ts

```diff
@@ -5,8 +5,14 @@
  */
 
 import { sortBy } from 'lodash';
+import { ContextFunctionSpec, Datatable } from '../types';
 
-export function sort() {
+interface Arguments {
+  by: string;
+  reverse: boolean;
+}
+
+export function sort(): ContextFunctionSpec<'sort', Datatable, Arguments, Datatable> {
```

Comment:
I see how `ContextFunctionSpec` slots in the type def for the arguments of `fn` in row 38 below, learning from it! I don't understand the part though where `ContextFunctionSpec` specifies a 3rd argument (`handlers`) for `fn` but for the linter, two arguments here (` fn: (context, args) => {`) seems to cause no concern,  not sure why

Reply:
Function arguments toward the end of the declaration need not be included if they aren't used.